docs/src/postmortem/2021-07-17.md

```diff
@@ -0,0 +1,114 @@
+# Post Mortem 2021-06-06
+
+## Incident summary
+
+For approximately 2 days the Nomad managed tasks on a-fsn-de were
+unavailable, including repository management tasks.  Resolution was
+and continues to be hampered by a distinct and ongoing outage.
+
+## Leadup
+
+At this time we do not know what caused the reboot of a-fsn-de,
+however we do know that a parallel and ongoing incidend was occuring
+at the same time.  Void uses a centralized authentication service (CAS) to
+manage access to our machines, and like many secure services this one
+relies on TLS certificates.  This certificate expired without being
+noticed which prevented what would have otherwise been a quick
+recovery of logging in and bouncing some services.
+
+Additionally, when the CAS is unavailable, we maintain a break-glass
+login capability for a handful of extremely trusted users (1:1 with
+the people that have access to the package signing key).  This access
+was discovered to be impaired by 1 developer's key being missing
+entirely, and one developer having failed to rotate their key.  The
+third developer was on vacation but was able to log in and rectify the
+keying problem.
+
+## Fault
+
+The Nomad outage was caused by an unexpected restart of a-fsn-de.
+When Nomad hosts reboot there is a known defect in that runit may
+bring up the services in a race that results in Nomad not being usable
+until services are restarted in a specific order.
+
+The unavailability of the CAS system means that we still cannot log in
+to all hosts normally.
+
+The issues with the break-glass keys caused the recovery of both the
+specific Nomad host and the CAS server to be slow.
+
+## Impact
+
+Publicly Visible:
+
+  * Repository signing unavailable
```

Comment:
Maybe more specific? "Signatures for packages that were built after Nomad were unavailable, even though the packages were added to repository data. This included `less`, which is installed on most machines and therefore made the failure highly visible".

Ah, you mention less below, but I think being explicit about the affected packages at least is reasonable.